types/react-alert/index.d.ts

```diff
@@ -1,22 +1,23 @@
-// Type definitions for react-alert 5.2
+// Type definitions for react-alert 7.0
 // Project: https://github.com/schiehll/react-alert
 // Definitions by: Yue Yang <https://github.com/g1eny0ung>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
```

Comment:
```suggestion
```
this line is effectively dead code now, we could remove this IMO

Reply:
Got it!

Reply:
Fixed in e18eb04.